pandas/tests/indexes/period/test_period.py

```diff
@@ -11,6 +11,36 @@
 from ..datetimelike import DatetimeLike
 
 
+class TestPeriodLevelMultiIndex(object):
+    # TODO: Is there a more appropriate place for these?
```

Comment:
Let's figure that out in this PR (at first look, this location *might* be fine).

BTW, your class name ins't entirely accurate IIUC.  Your first test doesn't involve a multi-index.

Reply:
Fair enough.  The class originally had several other tests before this PR was broken up into 3 pieces.  Have a suggested name?

Reply:
the 2nd should go in test_multi.py